src/Surfnet/StepupMiddleware/ApiBundle/Identity/Projector/RaListingProjector.php

```diff
@@ -85,4 +86,20 @@ public function applyIdentityAccreditedAsRaaEvent(IdentityAccreditedAsRaaEvent $
 
         $this->raListingRepository->save($raListing);
     }
+
+    public function applyRegistrationAuthorityInformationAmendedEvent(
+        RegistrationAuthorityInformationAmendedEvent $event
+    ) {
+        /** @var RaListing $raListing */
+        $raListing = $this->raListingRepository->find($event->identityId);
+
+        if (!$raListing) {
+            return;
```

Comment:
This would be a quite awkward situation, as in the RA the action will be triggered from an overview populated by the RAListings.... Exception worthy I'd say...